### ISSUE TITLE
Downloading Mull crashes the app

### ISSUE THREAD

**Issue by NatureHog** [Tobi823/ffupdater#239]
For whatever reason, attempting to download Mull Browser crashes the app. It seems to only affect Mull Browser, for some reason...

Stacktrace:
```
de.marmaro.krt.ffupdater.background.UnrecoverableBackgroundException: The background job fails due to an unrecoverable exception.
	at de.marmaro.krt.ffupdater.BackgroundJob$doWork$2.invokeSuspend(Unknown Source:53)
	at w3.a.resumeWith(Unknown Source:8)
	at j4.i0.run(Unknown Source:94)
	at n4.a$a.run(SourceFile:9)
Caused by: java.util.NoSuchElementException: Collection contains no element matching the predicate.
	at de.marmaro.krt.ffupdater.network.fdroid.FdroidConsumer.getLatestUpdate(Unknown Source:288)
	at de.marmaro.krt.ffupdater.network.fdroid.FdroidConsumer$getLatestUpdate$1.invokeSuspend(Unknown Source:12)
	at w3.a.resumeWith(Unknown Source:8)
	at j4.i0.run(Unknown Source:94)
	at m4.e.run(Unknown Source:11)
	at n4.j.run(Unknown Source:2)
	... 1 more
```
Device information:
| Key | Value |
| --- | --- |
| FFUpdater version | 77.2.0 (113) release |
| Device | CPH2399 (CPH2399EEA, OP557AL1, k6893v1_64) |
| Manufacturer | OnePlus (OnePlus) | 
| Supported ABIs | arm64-v8a, armeabi-v7a, armeabi |
| Android version | 12 (SDK: 31) |
| OS | dg02-pool03-kvm249, root, release-keys, 1660632738000 |

**Comment by CPacaud123** [comment]
I seem to be having the same issue.

Stacktrace:
```
de.marmaro.krt.ffupdater.background.UnrecoverableBackgroundException: The background job fails due to an unrecoverable exception.
	at de.marmaro.krt.ffupdater.BackgroundJob$doWork$2.invokeSuspend(Unknown Source:53)
	at w3.a.resumeWith(Unknown Source:8)
	at j4.i0.run(Unknown Source:94)
	at n4.a$a.run(SourceFile:9)
Caused by: java.util.NoSuchElementException: Collection contains no element matching the predicate.
	at de.marmaro.krt.ffupdater.network.fdroid.FdroidConsumer.getLatestUpdate(Unknown Source:288)
	at de.marmaro.krt.ffupdater.network.fdroid.FdroidConsumer$getLatestUpdate$1.invokeSuspend(Unknown Source:12)
	at w3.a.resumeWith(Unknown Source:8)
	at j4.i0.run(Unknown Source:94)
	at m4.e.run(Unknown Source:11)
	at n4.j.run(Unknown Source:2)
	... 1 more
```
Device information:
| Key | Value |
| --- | --- |
| FFUpdater version | 77.2.0 (113) release |
| Device | LG-H873 (lucye_global_ca, lucye, QC_Reference_Phone) |
| Manufacturer | lge (LGE) | 
| Supported ABIs | arm64-v8a, armeabi-v7a, armeabi |
| Android version | 9 (SDK: 28) |
| OS | LGEACI8R11, jenkins, release-keys, 1568718511000 |

**Comment by Tobi823** [comment]
is already fixed in version 77.3.0 

but F-Droid needs some time until the new update is available
